src/pretalx/agenda/templates/agenda/changelog_block.html

```diff
@@ -1,4 +1,7 @@
 {% load i18n %}
+{% if schedule.comment %}
+  <p>{{ schedule.comment }}</p>
```

Comment:
I'm thinking we should allow Markdown in the comment field, and then render it with our ``|rich_text`` template tag here.